### PR TITLE
[installer] add os to replace cluster without downtime

### PR DESCRIPTION
**Pull request by jzhupup**
@ErikJiang @Rei1010  麻烦 review 下

### CODE REVIEW

File: docs/zh/docs/install/best-practices/replace-os.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+      children:
+        kube_control_plane:
+          hosts:
+            centos-master1: null # 即为首个节点
+            centos-master2: null
+            centos-master3: null
+```
+
+#### 首个 Control Plane 节点替换
```
Comment (ErikJiang) [inline docs/zh/docs/install/best-practices/replace-os.md:99]: 应该先替换“非首 master ”再替换“首个 master”

File: docs/zh/docs/install/best-practices/replace-os.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+## 解决方案
+
+- 需要从 control plane 、worker 节点两个角度来处理
+- 迁移操作顺序：worker 节点迁移 → 非首个 control plane 节点迁移 → 首个 control plane 节点迁移
+- 在进行节点替换操作之前，安全起见，建议对集群的相关资源先行备份
+
+!!!note
+
+    本文档默认采用先增后删的操作顺序，实际场景可按需求调整操作顺序。
```
Comment (ErikJiang) [inline docs/zh/docs/install/best-practices/replace-os.md:25]: 与“先删后增”矛盾

File: docs/zh/docs/install/best-practices/replace-os.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+                centos-master3: null
+    ```
+
+    修改后：
+
+    ```yaml
+        children:
+            chakube_control_plane:
+            hosts:
+                centos-master2: null
```
Comment (ErikJiang) [inline docs/zh/docs/install/best-practices/replace-os.md:219]: 顺序有问题，应该是 2 -> 3 -> 1

**Comment by windsonsea** [comment]
See preview:
https://deploy-preview-4958--daocloud-docs.netlify.app/install/best-practices/replace-os

**Review by windsonsea — APPROVED** [review]
Seems it's ready for merge.

**Pull request merged**